plugins/updater/Cargo.toml

```diff
@@ -9,34 +9,42 @@ rust-version = { workspace = true }
 links = "tauri-plugin-updater"
 
 [package.metadata.docs.rs]
-rustc-args = [ "--cfg", "docsrs" ]
-rustdoc-args = [ "--cfg", "docsrs" ]
+rustc-args = ["--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "docsrs"]
 
 [build-dependencies]
-tauri-plugin = { workspace = true, features = [ "build" ] }
+tauri-plugin = { workspace = true, features = ["build"] }
 
 [dependencies]
 tauri = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 thiserror = { workspace = true }
 tokio = "1"
-reqwest = { version = "0.11", default-features = false, features = [ "json", "stream" ] }
+reqwest = { version = "0.11", default-features = false, features = [
+    "json",
+    "stream",
+] }
 url = { workspace = true }
 http = "0.2"
 dirs-next = "2"
 minisign-verify = "0.2"
-time = { version = "0.3", features = [ "parsing", "formatting" ] }
+time = { version = "0.3", features = ["parsing", "formatting"] }
 base64 = "0.21"
 percent-encoding = "2"
-semver = { version = "1", features = [ "serde" ] }
+semver = { version = "1", features = ["serde"] }
 futures-util = "0.3"
 tempfile = "3"
 zip = "0.6"
 tar = "0.4"
 
 [target."cfg(target_os = \"windows\")".dependencies]
 zip = { version = "0.6", default-features = false }
+windows = { version = "0.54", features = [
```

Comment:
I think we should use `windows-sys` instead for faster compile times and easier maintenance since it is not frequently updated.

Reply:
I'm not very familiar with those type conversions in `windows-sys`, how do we convert `OsStr` to `HSTRING` here?